### DISCUSSION TITLE
MediaRecorder Clear Blobs

### DISCUSSION THREAD

**Discussion by andrejhronco** [cables-gl/cables_docs#137] [! Welcome - readme first!]
I'm getting odd behavior from MediaRecorder where a previously recorded video is retained in the next recorded video. In other words; if you record videos back to back without patch refresh, `recordedBlobs` never gets cleared and just gets added to with each subsequent recording. If this is the case, should `recordedBlobs` after recording has stopped and the file has been downloaded?

**Comment by andrejhronco** [comment]
adding `recordedBlobs = [];` to `startRecording` works well.

**Comment by pandrr** [comment]
thanks for the report and fix, will update the op!
